config/application.rb

```diff
@@ -49,10 +56,14 @@ class Application < Rails::Application
       "zh-TW"]
     config.i18n.available_locales = available_locales
     config.i18n.fallbacks = {
+      "es-PE" => "es",
+      "eu"    => "es",
       "fr"    => "es",
       "gl"    => "es",
       "it"    => "es",
-      "pt-BR" => "es"
+      "pt-BR" => "es",
+      "ca"    => "val",
```

Comment:
I was sure this change will start a debate ...

> It looks like Catalan is 99% translated and Valencian is 67% translated, so the fallback probably makes more sense the other way around 🤔. What do you think?

Yes probably you are right, but after a deeper thinking about fallbacks for Spanish languages (Asturianu, Catalan, Euskara, Galego and Valencian) maybe the best option is to setup `es` as language fallback for all of them as Spanish translations are always complete and Spanish (`es`) is a co-official language on each language region. If you agree with this criteria the fallbacks for Spanish would change to:

* `ast` => `es` (I did not enabled Asturianu because is less than 60% translated)
* `ca` => `es`
* `eu` => `es`
* `gl` => `es`
* `val` => `es`

> By the way, it looks like these languages are ordered alphabetically by key 😄. So Occitan would go before Brazilian Portuguese.

Nice catch! 🙏

Reply:
Fine with me :smile:.

Reply:
Done!